views/jobRoleDetail.html

```diff
@@ -34,8 +34,8 @@ <h4 id="responsibilities"> Responsibilities</h4>
                     <p id="responsibilities">{{ jobRole.responsibilities }}</p>
 
                 
-                    <p id="capability"><b>Capability: </b>{{ jobRole.capability }}</p>
-                    <p id="ban"><b>Band: </b>{{ jobRole.band }}</p>
+                    <p id="capability"><b>Capability: </b>{{ jobRole.capabilityID }}</p>
+                    <p id="ban"><b>Band: </b>{{ jobRole.bandID }}</p>
```

Comment:
you shouldn't be showing IDs on the screen, instead get back the band and capability names and display here